.github/workflows/auto-diff-k8s-ci.yaml

```diff
@@ -128,12 +128,13 @@ jobs:
       - name: get latest kind node version
         run: |
           bash ./test/scripts/update-kindnode-version.sh ${{ env.MINIMUM_K8S_VERSION }} ${{ env.K8S_MATRIX_FILE_PATH }}
-          if [ $? -eq 0 ]; then
+          if [ $? -ne 0 ]; then
             echo "./test/scripts/update-kindnode-version.sh script failed"
             echo "SKIP_CREATE_PR=true" >> $GITHUB_ENV
-            exit 0
+            exit 1
```

Comment:
this should not exit 1 to terminate the CI. Cause the kind version does not matter so much for the true purpose of this CI

Reply:
done